app/models/conference.rb

```diff
@@ -82,6 +82,16 @@ def date_range_string
   end
 
   ##
+  # Check if there is any event in the program scheduled.
+  #
+  # ====Returns
+  # * +false+ ->  If there aren't schedules events
+  # * +true+ -> If there are schedules events
+  def scheduled?
```

Comment:
because it was called only once as `conference.program.schedule?` and moving it I can write `conference.schedule?` I think is better.